openhexa/cli/api.py

```diff
@@ -241,6 +241,7 @@ def upload_pipeline(config, pipeline_directory_path: str):
         zipFile.seek(0)
 
     base64_content = base64.b64encode(zipFile.read()).decode("ascii")
+    parameters = [spec.__dict__ for spec in parameters_specs]
```

Comment:
You should use https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict